invenio_rdm_records/services/config.py

```diff
@@ -231,6 +232,7 @@ class RDMRecordServiceConfig(RecordServiceConfig, ConfiguratorMixin):
     schema_access_settings = AccessSettingsSchema
     schema_secret_link = SecretLinkSchema
     schema_grant = GrantSchema
```

Comment:
this needs to be removed?

Reply:
No, it is still used in update draft actions